codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/protocol/MakeOperationGenerator.kt

```diff
@@ -69,7 +69,7 @@ open class MakeOperationGenerator(
         val takesOwnership = bodyGenerator.bodyMetadata(shape).takesOwnership
         val mut = customizations.any { it.mutSelf() }
         val consumes = customizations.any { it.consumesSelf() } || takesOwnership
-        val self = "self".letIf(mut) { "mut $it" }.letIf(!consumes) { "&$it" }
+        val self = "self".letIf(mut) { "mut $it" }.letIf(!consumes) { "$it" }
```

Comment:
I think the last branch can be removed since it doesn't actually change the input

Reply:
You are right, fixed!